src/OrchardCore.Modules/Orchard.Autoroute/Indexes/AutoroutePartIndex.cs

```diff
@@ -18,7 +18,7 @@ public override void Describe(DescribeContext<ContentItem> context)
                 .Map(contentItem =>
                 {
                     var path = contentItem.As<AutoroutePart>()?.Path;
-                    if (!String.IsNullOrEmpty(path))
+                    if (!String.IsNullOrEmpty(path) && (contentItem.Published || contentItem.Latest))
```

Comment:
I'm able to alter AutoroutePartIndex in a migration, 
```
public int UpdateFrom1()
        {
            SchemaBuilder.AlterTable(nameof(AutoroutePartIndex), table => table
                .AddColumn<bool>("Published")
            );

            return 2;
        }
```

but I can't find any way to update data in the index table. I see there is an issue in the [yessql repository](https://github.com/sebastienros/yessql/issues/27) that tracks exactly the same probelm ... is there any workaround?